js/ui/grid_core/ui.grid_core.virtual_scrolling.js

```diff
@@ -1431,7 +1431,7 @@ export const virtualScrollingModule = {
                                 this.updateItems({
                                     repaintChangesOnly: true,
                                     needUpdateDimensions: true,
-                                    useProcessedItemsCache: true,
+                                    useProcessedItemsCache: !this.getController('editing')?.getChanges()?.length,
```

Comment:
Let's move these complicated conditions to a separate variables

E.g
```typescript
const loadingItemsStarted = this._loadItems(checkLoading, !viewportIsNotFilled);
// not perfect variable name feel free to name it another way :)
const isNeedToUpdateItems = !loadingItemsStarted && !(this._isLoading && checkLoading) && !checkLoadedParamsOnly;


if(isUpdateItemsNeeded) {
  const editingHasChanges = !!this.getController('editing')?.getChanges()?.length
  this.updateItems({
    repaintChangesOnly: true,
    needUpdateDimensions: true,
    useProcessedItemsCache: true,
    useProcessedItemsCache: !editingHasChanges,
    cancelEmptyChanges: true
  });
}
```

Reply:
Applied the suggested changes.
Not a fan of triple negating a variable though so i've changed naming a bit